sqlglot/dialects/drill.py

```diff
@@ -148,4 +148,7 @@ class Generator(generator.Generator):
             e: f"DATE_ADD(CAST({self.sql(e, 'this')} AS DATE), {self.sql(exp.Interval(this=e.expression, unit=exp.var('DAY')))})",
             exp.TsOrDiToDi: lambda self,
             e: f"CAST(SUBSTR(REPLACE(CAST({self.sql(e, 'this')} AS VARCHAR), '-', ''), 1, 8) AS INT)",
+            exp.Levenshtein: unsupported_args("ins_cost", "del_cost", "sub_cost", "max_dist")(
+                rename_func("LEVENSHTEIN_DISTANCE")
+            ),
```

Comment:
Let's move that to it's previous position so the dictionary is alphabetically sorted (between `exp.ILike` and `exp.PartitionedbyProperty`)